packages/adp-tooling/templates/rta/common/v4-table-action.xml

```diff
@@ -0,0 +1,5 @@
+<core:FragmentDefinition  xmlns:core='sap.ui.core' xmlns='sap.m'>
+   <actiontoolbar:ActionToolbarAction xmlns:actiontoolbar="sap.ui.mdc.actiontoolbar" id="<%- ids.customToolbarAction%>" >
+        <Button xmlns:m="sap.m" id="<%- ids.customActionButton %>" visible="true" text="Click here" />
```

Comment:
Initially I thought that the Quick Action did not work, but then it turned out that the action name was just not clear enough to indicate that it was just added. We should use a text for the button that is a bit more distinctive and can be easier to understand that it is something that was just added.

```suggestion
        <Button xmlns:m="sap.m" id="<%- ids.customActionButton %>" visible="true" text="New Action" />
```